contracts/PeriodicTokenVesting.sol

```diff
@@ -35,6 +35,9 @@ contract PeriodicTokenVesting is OwnableUpgradeable, PausableUpgradeable {
     /// @dev The duration in seconds of a vesting period.
     uint256 private periodDuration;
 
+    /// @dev The duration in seconds of the cliff.
+    uint256 private cliffDuration;
```

Comment:
I would name it `cliff`

Reply:
Same as with period, cliff alone as a uint256 could be the timestamp when the cliff ends (like the old contract) or the duration. 

Appending the duration gives the variable a more precise meaning of what it is used for.